### ISSUE TITLE
test:postprocessors/multi_period_averaging.receiver periodically fails

### ISSUE THREAD

**Issue by lindsayad** [shannon-lab/zapdos#253]
example at https://civet.inl.gov/job/2364929/ for https://github.com/idaholab/moose/pull/27810

**Comment by lindsayad** [comment]
@gsgall would you want to look at this or know someone who would?

**Comment by gsgall** [comment]
@lindsayad I can take a look at it

**Comment by gsgall** [comment]
I am not sure I understand what the example you provided is showing me about the Zapdos App, the Civet job and moose issue don't seem to be related to Zapdos, are these changes MOOSE changes breaking this Post-processor?

**Comment by lindsayad** [comment]
No, I think that the test is just brittle. Sometimes it passes and sometimes it doesn't

**Comment by cticenhour** [comment]
Addressed in #255
